managed/src/main/java/com/yugabyte/yw/controllers/CustomerConfigController.java

```diff
@@ -39,6 +43,15 @@ public Result create(UUID customerUUID) {
     if (errorJson.size() > 0) {
       return ApiResponse.error(BAD_REQUEST, errorJson);
     }
+    if (formData.get("name").asText().equals("S3") &&
+      formData.get("data").get("AWS_ACCESS_KEY_ID") != null) {
+      String region = Region.getByProvider(Provider.get(customerUUID, Common.CloudType.aws).uuid)
```

Comment:
This doesn't work if we don't have the AWS provider configured (will raise NPE). The page with Backup settings displays all the settings despite the fact that some of providers are not configured yet.

Reply:
He should not be using the region from provider though.  He should use the s3Region he got in the formData.

Reply:
Actually we don't have **s3Region** in form Data. 
The idea here was to take a backup for that we needed **universe** and to create a universe we need a provide right? 
So one provider should be configured to take a backup that could be of any type instead of hard coding it to AWS.
@sb-yb  Any thoughts on this?